postcode_lookup/template_sorter.py

```diff
@@ -306,7 +311,17 @@ def __init__(
             "timetable": self.timetable,
         }
 
-        enabled_sections = [BallotSection(**section_kwargs)]
+        # TODO: Also make this false for London
+        self.has_parishes = country != Country.NORTHERN_IRELAND
```

Comment:
I will circle back to this but I need to expose the `identifiers` field in the schema first so lets not block on it